packages/ember-self-focused/addon/components/self-focused.js

```diff
@@ -1,19 +1,28 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
+import { getOwner } from '@ember/application';
+import { computed } from '@ember/object';
 import layout from '../templates/components/self-focused';
 
 export default Component.extend({
   layout,
   focusManager: service(),
   classNames: ['self-focused'],
+  fastboot: computed(function() {
+    return getOwner(this).lookup('service:fastboot');
+  }),
 
   didInsertElement() {
     this._super(...arguments);
-    this.get('focusManager').didInsertElement(this.element);
+    if (!this.get('fastboot.isFastBoot')) {
+      this.get('focusManager').didInsertElement(this.element);
+    }
   },
 
   didRender() {
     this._super(...arguments);
-    this.get('focusManager').didRenderElement(this.element);
+    if (!this.get('fastboot.isFastBoot')) {
```

Comment:
Same as above, `didRender` is only called during "interactive" renders, aka not in fastboot.

Reply:
Done!